libs/types-common/src/Data/Json/Util.hs

```diff
@@ -157,6 +157,18 @@ class ToJSONObject a where
 instance ToJSONObject A.Object where
   toJSONObject = id
 
+-----------------------------------------------------------------------------
+-- Aeson Object
+
+instance S.ToParamSchema A.Object where
+  toParamSchema _ =
+    mempty & S.type_ ?~ S.SwaggerString
+
+instance ToSchema A.Object where
+  schema =
+    named "Object" $
+      id .= jsonObject
```

Comment:
```suggestion
objectToParamSchema :: ...
objectToParamSchema _ = mempty & S.type_ ?~ S.SwaggerString

objectSchema :: ...
objectSchema = named "Object" $ id .= jsonObject      
```

i think this could work for `Schema`, not sure about `ParamSchema`.  if it works, it'll reduce the need for orphans, and the places where we use this are pretty few.

Reply:
Well, the issue isn't the Swagger of `Object`, it's that `List1 Object` has to fulfill this constraint: `instance ToSchema a => ToSchema (List1 a)`

I don't know a good way out: One the one hand we'll likely want to show the Swagger of the payload of `List1`. On the other hand, this orphan is a bit unfortunate... :roll_eyes: 

https://github.com/wireapp/wire-server/pull/3003/files#diff-04dac01f8df7b94a25e2d4350eb8e448d47ea92bfb5fe15fa0ba7db8720334aeR67

Reply:
ah, i see.  then orphans it is i guess!  :+1: